README.md

```diff
@@ -6,7 +6,6 @@
 [![Docker status][github-docker-badge]][github-docker-link]
 [![Release status][github-release-badge]][github-release-link]
 [![GoDev][godev-badge]][godev-link]
-[![GitBook][gitbook-badge]][gitbook-link]
```

Comment:
Can you add a Docusaurus badge that link to our Docs site from the top of the README?

Reply:
Nice ;)